src/chart/MultiChart.js

```diff
@@ -74,7 +74,9 @@
     MultiChart.prototype._mapChartTypes = [
         { id: "CHORO_USSTATES", display: "US State Choropleth", widgetClass: "map_ChoroplethStates" },
         { id: "CHORO_USCOUNTIES", display: "US County Choropleth", widgetClass: "map_ChoroplethCounties" },
-        { id: "CHORO_COUNTRIES", display: "Counrty Choropleth", widgetClass: "map_ChoroplethCountries" }
+        { id: "CHORO_COUNTRIES", display: "Country Choropleth", widgetClass: "map_ChoroplethCountries" },
+        { id: "GOOGLE_MAP", display: "Goole Map", widgetClass: "map_GMapLayered" },
+        { id: "OPENSTREET", display: "Goole Map", widgetClass: "map_OpenStreet" }
     ].map(function (item) { item.family = "map"; return item; });
```

Comment:
Misspelled "Google Map". Also, should both be "Google Map" or should the second be "Open Street"?